packages/server/src/registration.js

```diff
@@ -43,7 +43,7 @@ exports.parseRegisterRequest = (body) => {
     };
 };
 
-exports.generateRegistrationChallenge = ({ relyingParty, user } = {}) => {
+exports.generateRegistrationChallenge = ({ relyingParty, user, attestation } = {}) => {
```

Comment:
```suggestion
exports.generateRegistrationChallenge = ({ relyingParty, user, attestation = 'redirect' } = {}) => {
```

Reply:
Very true. I have committed that change to the branch.